utils/wasm-proc/src/main.rs

```diff
@@ -173,14 +173,15 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         }
 
         let file = PathBuf::from(file);
-        let res = gear_wasm_builder::optimize::optimize_wasm(file.clone(), "s", true)?;
-
-        log::info!(
-            "wasm-opt: {} {} Kb -> {} Kb",
-            res.dest_wasm.display(),
-            res.original_size,
-            res.optimized_size
-        );
+        // Issue (#1971)
+        // let res = gear_wasm_builder::optimize::optimize_wasm(file.clone(), "s", true)?;
+
+        // log::info!(
```

Comment:
Doesn't `wasm-opt` work at all?

Reply:
doesn't work for me locally

Reply:
what version did you try?

Reply:
> what version did you try?

wasm-opt version 111

Reply:
v111 doesn't work for me too, but v110 does.